aws/cloudformation/standalone/gen_ai_curriculum/README.md

```diff
@@ -0,0 +1,33 @@
+Deploys AWS resources for the Gen AI Curriculum. These include SageMaker models, endpoint configurations, and endpoints.
+
+### Usage:
+```
+./deploy_gen_ai_curriculum_stack [-e | --environment] [-d | --dry-run] [-i | --image-uri] [-t | --huggingface-token]
+```
```

Comment:
So for test and production deployments, the correct command is:

`./deploy_gen_ai_curriculum_stack -e test`

and

`./deploy_gen_ai_curriculum_stack -e production`

with no additional options?

Reply:
My goal is for whatever's deployed to be 100% defined by code. By including these optional parameters, we introduce an unknown: "What did the engineer who deployed this last time specify for these options?"

So I'd like to remove the unknown by explicitly stating what those options should be for a test or production deployment, or remove them as options and go purely based on the contents of the repo.

Reply:
Technically no options are required; the command is just `./deploy_gen_ai_curriculum_stack` and the script will prompt for any missing parameters. Hopefully the instructions in the script aren't too ambiguous (i.e. shouldn't require knowledge of what an engineer did last time)?

Once the TODO for setting up the token in secrets manager is resolved, the only parameters that user has to choose are environment and dry-run (which are both prompted for). The rest will work with defaults.

Reply:
Token has been added to secrets manager, so it will now be fetched automatically (no need to supply one unless a user wants to use a custom token).